server/src/main/java/org/elasticsearch/snapshots/RestoreService.java

```diff
@@ -165,8 +166,7 @@
 
     public RestoreService(ClusterService clusterService, RepositoriesService repositoriesService,
                           AllocationService allocationService, MetadataCreateIndexService createIndexService,
-                          MetadataIndexUpgradeService metadataIndexUpgradeService, ClusterSettings clusterSettings,
-                          ShardLimitValidator shardLimitValidator) {
+                          MetadataIndexUpgradeService metadataIndexUpgradeService, ShardLimitValidator shardLimitValidator) {
```

Comment:
Aren't the `ClusterSettings` needed anymore?

Reply:
Nah just dead code, sorry for mixing this in here just saw it and it annoyed me :)